firecloud/fiss.py

```diff
@@ -1226,12 +1228,29 @@ def mop(args):
         # Check output produces a string in Py2, Bytes in Py3, so decode if necessary
         if type(bucket_files) == bytes:
             bucket_files = bucket_files.decode()
+        
+        #List of user submission within this workspace. To ensure deletion of files in the submission directories only
+        user_submission = fapi.list_submissions(args.project, args.workspace)
+        
+        for item in user_submission.json():
+            sub_id = item['submissionId']
+            submission_ids.append(sub_id)
+        
+        bucket_files = set(bucket_files.strip().split('\n'))
+        for submission_id in submission_ids:
+            for bucket_file in bucket_files:
+                if submission_id in bucket_file:
+                     filtered_bucket_files.add(bucket_file)
+
+        
+        bucket_files = filtered_bucket_files
```

Comment:
Similarly more "pythonic" version, avoiding the need for defining `filtered_bucket_files` altogether
```python
bucket_files = set(bucket_file for bucket_file in bucket_files if bucket_file.split('/', 4)[3] in submission_ids)
```